src/apps/code-editor/src/app/components/FileActions/components/Delete/DeleteDialogue.js

```diff
@@ -44,53 +47,62 @@ export const DeleteDialogue = memo(function DeleteDialogue({
   }, [dispatch, fileZUID, status, onClose, history]);
 
   return (
-    <Dialog open={isOpen} fullWidth maxWidth={"xs"} onClose={onClose}>
-      <DialogTitle>
-        <Box
-          sx={{
-            backgroundColor: "red.100",
-            borderRadius: "100%",
-            width: "40px",
-            height: "40px",
-            display: "flex",
-            justifyContent: "center",
-            alignItems: "center",
-          }}
-        >
-          <DeleteRounded color="error" />
-        </Box>
-        <Stack
-          mt={1.5}
-          display="flex"
-          justifyContent="flex-start"
-          columnGap={1}
-        >
-          <Typography variant="inherit" fontWeight={700}>
-            Delete File:
+    <ThemeProvider theme={theme}>
```

Comment:
Why are we adding a different `ThemeProvider` here? I see 2 more ThemeProvider declarations aside from this one as well. I thought our goal here is to generally avoid using multiple ThemeProviders on different locations and just use a top-level one.